cmd/tink-worker/tink/tink.go

```diff
@@ -0,0 +1,70 @@
+package tink
+
+import (
+	"context"
+	"reflect"
+
+	"github.com/pkg/errors"
+	pb "github.com/tinkerbell/tink/protos/workflow"
+)
+
+type Wdata struct {
```

Comment:
What's the purpose behind having this `Wdata` struct? would `WorkflowData` not satisfy on its own?